commons-geometry-euclidean/src/main/java/org/apache/commons/geometry/euclidean/twod/Vector2D.java

```diff
@@ -565,5 +571,11 @@ public Vector2D normalize() {
         public Vector2D withNorm(final double mag) {
             return multiply(mag);
         }
+
+        /** {@inheritDoc} */
+        @Override
+        public UnitVector negate() {
+            return new UnitVector(-super.x, -super.y);
```

Comment:
Can we remove the super here? Same in the other classes.

Reply:
Removing super doesn't work, as the compiler thinks we're trying to access VectorXD.x statically, and using `this.x` doesn't work, as the compiler complains about VectorXD.x being private.

Reply:
I will defer to the compiler, in that case:-)

Reply:
Won't this be inlined to the same thing and avoids using the lesser known `super` keyword:
```
return new UnitVector(-getX(), -getY());
```

Reply:
I believe it would. That would be more consistent with our other code, too.